.github/workflows/lint_mypy.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     - uses: actions/setup-python@v1
       with:
         python-version: '3.x'
-    - run: python -m pip install pylint
+    - run: python -m pip install pylint==2.4.4
```

Comment:
Probably better to just use `pylint!=2.5.0` since upstream is aware of this bug and will likely fix it next release.